### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by LeOndaz** [maxence-charriere/go-app#924]
Do we have any people who have access to this? In such case, I’m willing to contribute

**Comment by oderwat** [comment]
What do you mean? This is an active repository in steady development by @maxence-charriere and there is a v10 release in the making. We use v9 for production level applications for quite some time now. Having a stable release does not mean something is death. It may just be so good that you do not need steady fixes to the code base :)

**Comment by oderwat** [comment]
@LeOndaz Well, there was a similar query just yesterday. I am not the maintainer anyway. I think you should check out #875.

@maxence-charriere It looks like the people just check master and may even bail without writing something. Maybe it would be a good idea to develop in master or change the default branch when starting a new release cycle? It may also be good to close some of the ancient issues and have a FAQ for some of the frequent problems people run into. Some of them are answered inside already close issues. I guess this will probably need more time than you can spend on that right now. I prefer if you use your time to finish v10, and also have to do quite a lot of other stuff in my company. But maybe you could start a little FAQ and link it from the README, so people can contribute to that using PRs. Or we start a Wiki for that?

**Comment by maxence-charriere** [comment]
Yes, definitely. I will soon release v10 and see how those things can be improved.

**Comment by gepengscu** [comment]
I'm looking forward to the release of v10! What are the new features in v10?

**Comment by shindakioku** [comment]
@maxence-charriere 
I apologize for mentioning you here. By the way, I'm interested in helping you with code as a developer. However, I don't see any specific issues that I can address, and I'm not entirely clear on your work for the new branch (v10). I mean, you're implementing a new version without providing any information or updates. I understand that you're working alone and why you might not be doing so. Additionally, I understand that there might not be much you can do about it.

**Comment by maxence-charriere** [comment]
@gepengscu I rewrote the engine to be more maintainable and efficient. Al also introduced some API changes, like how `app.If` is working, and some fixing related to issues I got.

@shindakioku, thank you for offering your contribution.
To be honest it can be frustrating to contribute to go-app. Since it is the backbone of Murlok.io, I'm very picky about what gets in, and not a lot of PR make the cut. I do reuse some code or take inspiration from them.
I guess it is like in a work environment, I get to be convinced that something is necessary and has good value. Issues might be a better place to start contributing since there is little coding investment there.
